sql/moz-fx-data-shared-prod/fenix_derived/meta_attribution_country_counts_v1/query.sql

```diff
@@ -8,15 +7,20 @@ SELECT
       THEN 'Firefox Browser for Android'
     WHEN (metrics.string.meta_attribution_app = '697946762208244')
       THEN 'Firefox Nightly for Android'
-    ELSE CAST(NULL AS STRING)
+    ELSE NULL
   END AS meta_attribution_app,
   normalized_channel,
   metadata.geo.country AS country,
   COUNT(*) AS ping_count
 FROM
   fenix.first_session
 WHERE
-  DATE(submission_timestamp) = @submission_date
+  DATE(submission_timestamp) >=
+  {% if is_init() %}
+    '2023-11-01'
+  {% else %}
+    @submission_date
+  {% endif %}
```

Comment:
`DATE(submission_timestamp) >= @submission_date` is a change in behavior and won't work properly if this is run for any historical dates.
```suggestion
  {% if is_init() %}
    DATE(submission_timestamp) >= '2023-11-01'
  {% else %}
    DATE(submission_timestamp) = @submission_date
  {% endif %}
```